### PR TITLE
Add JS example requesting device with higher limit

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR adds a JS example requesting device with higher limit to help developers.

@toji Please review.

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -2810,6 +2810,19 @@ dictionary GPUDeviceDescriptor
     </pre>
 </div>
 
+<div class=example>
+    Requesting a {{GPUDevice}} with a higher {{supported limits/maxColorAttachmentBytesPerSample}} limit:
+
+    <pre highlight=js>
+        const gpuAdapter = await navigator.gpu.requestAdapter();
+
```
Comment (toji) [inline spec/index.bs:2818]: For the sake of promoting best practices, can you add some code here that checks to see if the adapter supports the desired limit first? We don't want to encourage blindly setting required limits without giving the user feedback when something fails.

Could be something as simple as:

```js
if (gpuAdapter.limits.maxColorAttachmentBytesPerSample < 64) {
  // When the desired limit isn't supported, take action to either fall back to a code path that does not
  // require the higher limit or notify the user that their device does not meet minimum requirements.
}
```

**Review by toji — APPROVED** [review]
LGTM once one change has been made.

File: spec/index.bs (hunk 1 above)
Comment (beaufortfrancois) [reply spec/index.bs:2818]: Done.

**Review by beaufortfrancois — COMMENTED** [review]
@toji  If that still looks good to you, please merge.

**Pull request merged**